spec.emu

```diff
@@ -1019,8 +1019,8 @@ contributors: Ron Buckton, Ecma International
       </dl>
       <emu-alg>
         1. If _method_ is not present then,
-          1. If _V_ is *null* or *undefined*, return ~unused~.
-          1. If Type(_V_) is not Object, throw a *TypeError* exception.
+          1. If _V_ is either *null* or *undefined* and _hint_ is ~sync-dispose~, then
+            1. Return ~unused~.
```

Comment:
I'd appreciate a NOTE capturing the rationale here.

Reply:
I've added a NOTE, please let me know if it is clear enough.

Reply:
Yep, note looks good, thanks!